setup.py

```diff
@@ -47,11 +42,19 @@
     include_package_data=True,
 
     # Allows `setup.py test` to work correctly with pytest
-    setup_requires=['numpy>=1.19.5'] + pytest_runner,
+    setup_requires=[] + pytest_runner,
 
     # Additional entries you may want simply uncomment the lines you want and fill in the data
     # url='http://www.my_package.com',  # Website
-    # install_requires=[],              # Required packages, pulls from pip if needed; do not use for Conda deployment
+    install_requires = [
```

Comment:
move the install_requires bits here (I don't have an explanation, I _think_ that's just how it works)